docs/introduction/blackfire-profiling.md

```diff
@@ -0,0 +1,81 @@
+# Blackfire profiling
+
+[TOC]
+
+## Introduction
+
+Blackfire is a powerful tool for profiling and debugging that helps you to understand the performance of your application.
+In the Shopsys Platform, Blackfire is preinstalled for local development, allowing developers to easily profile and optimize their code.
+
+## Prerequisites
+
+Before you can start using Blackfire in the Shopsys Platform, you need to have the following:
+
+-   Blackfire account
+    -   You can create an account on the [Blackfire website](https://blackfire.io/).
+-   Blackfire browser extension installed (optional)
+    -   The extension is available for [Google Chrome](https://docs.blackfire.io/integrations/browsers/chrome) and [Mozilla Firefox](https://docs.blackfire.io/integrations/browsers/firefox).
```

Comment:
```suggestion
-   [Blackfire browser extension](https://docs.blackfire.io/integrations/browsers/index) installed (optional)
```

I think this way it is better, so we do not have to update our docs if they introduce extension for other browsers.